### PR TITLE
[#2] create Exception handler class

### PR DESCRIPTION
**Pull request by bohdanleliuk**

### CODE REVIEW

**Comment by Styopa-91** [comment]
good job

File: src/main/java/com/bohdanllk/dao/HotkeyDAOImpl.java [hunk 1]
```diff
@@ -1,37 +1,27 @@
...
-        return list;
-    }
-
+    //todo delegate to framework
     public void addList(List<Hotkey> hotkeyList) {
         Session session = getSession();
         for (int i = 0; i < hotkeyList.size(); i++) {
             session.persist(hotkeyList.get(i));
-            if (i % 10 == 0) {
+            if (i % BATCH_SIZE == 0) {
```
Comment (nromanen) [inline src/main/java/com/bohdanllk/dao/HotkeyDAOImpl.java]: delete unnecessary iteration

Comment (bohdanleliuk) [reply src/main/java/com/bohdanllk/dao/HotkeyDAOImpl.java]: okey

File: src/main/java/com/bohdanllk/controller/AppController.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
 @RequestMapping("/app")
 public class AppController {
 
-    final AppService appService;
+    private final AppService appService;
```
Comment (petrokrechunyak) [inline src/main/java/com/bohdanllk/controller/AppController.java:16]: Usually it non-final

**Pull request merged**